tests/acceptance/components/flexberry-objectlistview/folv-column-width-customization-test.js

```diff
@@ -0,0 +1,110 @@
+import Ember from 'ember';
+import { executeTest } from './execute-folv-test';
+import { loadingList} from './folv-tests-functions';
+
+executeTest('column width custom', (store, assert, app) => {
+
+  let path = 'ember-flexberry-dummy-suggestion-list';
+  Ember.run(() => {
+    let done = assert.async();
+    visit(path);
+    andThen(function() {
+      assert.equal(currentPath(), path);
+      let $defaultConfig = Ember.$('.remove.circle.icon');
+      loadingList($defaultConfig[0], '.object-list-view-container', 'table.object-list-view tbody tr').then(($list) => {
+        assert.ok($list);
+        let $configButton = Ember.$('.ui.button')[6];
+        let done1 = assert.async();
+
+        // Loading config window
+        loadingList($configButton, '.object-list-view-container', 'table.object-list-view tbody tr').then(($list) => {
+          assert.ok($list);
+
+          // Retrieve component & it's inner <input>.
+          let $component = Ember.$('.flexberry-checkbox.ui.checkbox');
+
+          assert.ok($($component).hasClass('checked'), true, 'Width config is ON');
+          assert.notOk($($component).hasClass('disabled'), true, 'Width values is ON');
```

Comment:
The functions [`assert.ok`](https://api.qunitjs.com/assert/ok) and [`assert.notOk`](https://api.qunitjs.com/assert/notOk) have only two parameters.